requirements.txt

```diff
@@ -1,2 +1,4 @@
-github3.py==1.0.0a4
+pystache
+requests
```

Comment:
Isn't `github3.py` pulling in `requests` already. At least according to their `setup.py` file, are they depending on `requests.`

Reply:
Yes, it is, but I thought, requiring it explicitly is right. In case we get rid of github3.py, for example.
Should I remove it?

Reply:
I don't think that we get rid of github3.py as large parts of the code depends on it at the moment.

Reply:
We should always mention all dependencies that we directly rely on. If we import requests, we should list requests.